great_expectations/data_context/store/checkpoint_store.py

```diff
@@ -195,7 +195,7 @@ def create(self, checkpoint_config: CheckpointConfig) -> Optional[DataContextKey
         # Make two separate requests to set and get in order to obtain any additional
         # values that may have been added to the config by the StoreBackend (i.e. object ids)
         ref: Optional[GeCloudResourceRef] = self.set(key, checkpoint_config)  # type: ignore[func-returns-value]
-        if ref:
+        if ref and not isinstance(ref, bool):
```

Comment:
This patch was to address an exception that was being thrown only when updating an existing Checkpoint, using `context.add_checkpoint` (see update checkpoint notebook cell).

For some reason, `ref` was coming back as a `bool` only when updating.

Reply:
same as comment above